docs/io/unified.rst

```diff
@@ -269,6 +269,72 @@ FITS header (for example, keyword comments are dropped).
 
 .. _fits_astropy_native:
 
+
+TDISPn Keyword
+^^^^^^^^^^^^^^
+
+TDISPn FITS keywords will map to and from the `~astropy.table.Column` ``format``
+attribute if the display format is convertible to and from a Python display
+format. Below are the rules used for both conversion directions.
+
+TDISPn to Python Format String
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TDISPn format characters are defined in the table below.
+
+============   ================================================================
+   Format                              Description
+============   ================================================================
+Aw             Character
+Lw             Logical
+Iw.m           Integer
+Bw.m           Binary, integers only
+Ow.m           Octal, integers only
+Zw.m           Hexadecimal, integers only
+Fw.d           Floating-point, fixed decimal notation
+Ew.dEe         Floating-point, exponential notation
+ENw.d          Engineering; E format with exponent multiple of three
+ESw.d          Scientific; same as EN but non-zero leading digit if not zero
+Gw.dEe         General; appears as F if significance not lost, also E
+Dw.dEe         Floating-point, exponential notation
```

Comment:
As mentioned, I think for this case the exponentials may appear as `D` (at least in fortran, not sure about fits...)